commands/Eval.cs

```diff
@@ -67,6 +100,38 @@ public static void Info(IUser client)
                     l.ForEachUser(x => client.Print(Template.Text(Category.Info, 1).Replace("+n", x.Name).Replace("+v", x.Vroom.ToString()).Replace("+i", "linked")));
                 });
         }
+        
+        public static void screen(IUser admin, string name)

```

Comment:
1.) You assume C drive
2.) You assume sb0t.exe will always be called sb0t.exe in fact that is how you can run multiple instances of sb0t on same pc with no conflict is to rename the sb0t.exe to something else

What if sb0t is ran on a dedicated server or vps without a chat client being open you would have just screen shots of the desktop without any real context.  If the host is doing online banking I'm not sure they would want a screen shot of their bank account on their PC either